core/drivers/clk/sam/phy-sama7-utmi-clk.c

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
please reconsider the license terms for this contribution.

Reply:
updated to `BSD-2-Clause`